pkg/config/config_unix.go

```diff
@@ -0,0 +1,16 @@
+// +build !windows
+
+package config
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+)
+
+func pathOwnedByRoot(fi os.FileInfo, path string) error {
+	if fi.Sys().(*syscall.Stat_t).Uid != 0 || fi.Sys().(*syscall.Stat_t).Gid != 0 {
```

Comment:
Can you perform the type cast in the form of `if v, ok := whatever` and check the `ok` value?

Reply:
This was the existing code and I just moved it. Happy to make the changes.

Reply:
Sorry. If you dont' mind. It'd be safer with the change.